### PR TITLE
LPS-88060 [TS]

### PR DESCRIPTION
**Pull request by jcampoy**
Hi @Preston-Crary 

thanks for taking care of the fix, but if you do not add this commit existing portlets will lost their current customized configuration, what it is an important regression bug. I have tested the issue and check that portlets lost their configuration. Also I have tested this fix and works fine.

Could you take a look?

Thank you very much and best regards,
Jose

### CODE REVIEW

**Comment by jcampoy** [comment]
ci:test:sf

**Comment by jcampoy** [comment]
ci:test:relevant

**Comment by jcampoy** [comment]
ci:test:sf

File: portal-impl/src/com/liferay/portal/layoutconfiguration/util/velocity/TemplateProcessor.java [hunk 1]
```diff
@@ -212,11 +212,29 @@ public String processPortlet(
 			if (!layoutTypePortlet.hasPortletId(portletId, true) &&
 				!layout.isPortletEmbedded(portletId, layout.getGroupId())) {
 
+				ModifiableSettings currentSettings =
+					SettingsFactoryUtil.getSettings(
+						new PortletInstanceSettingsLocator(
+							layout, portletId)).getModifiableSettings();
+
 				PortletPreferencesFactoryUtil.getLayoutPortletSetup(
```
Comment (Preston-Crary) [inline portal-impl/src/com/liferay/portal/layoutconfiguration/util/velocity/TemplateProcessor.java]: Instead reading the settings twice and re-modifying them, can you update how getLayoutPortletSetup() is used to fix this problem?

From your changes it's not clear what part of getLayoutPortletSetup() is changing the settings.  If you can't figure out a way to update the how getLayoutPortletSetup() is used to fix the problem, provide me with some steps so I can.

Comment (jcampoy) [reply portal-impl/src/com/liferay/portal/layoutconfiguration/util/velocity/TemplateProcessor.java]: Hey @Preston-Crary , tomorrow I will resend you a fix with your suggestions. Thank you!

Comment (jcampoy) [reply portal-impl/src/com/liferay/portal/layoutconfiguration/util/velocity/TemplateProcessor.java]: https://github.com/Preston-Crary/liferay-portal/pull/708

**Pull request closed without merging**